ipfs-pinning-service.yaml

```
@@ -108,6 +108,27 @@ Pinning services are encouraged to add support for additional features by levera
 While these attributes can be vendor-specific, we encourage the community at large to leverage these `meta` attributes as a sandbox to come up with conventions that could become part of future revisions of this API.
 
 
+# Pagination and filtering
+
+Pin objects can be listed by executing `GET /pins` with optional parameters:
+
+
+- When no filters are provided, the endpoint will return a small batch of 10 most
+  recently created items, from the latest to the oldest.
+
+- The number of returned items can be adjusted with `limit` parameter
+  (implicit default is 10).
+
+- If the value in `PinResults.count` is bigger than the length of
+  `PinResults.results`, the client can infer there are more results that can be queried.
+
+- To read more items, pass the `before` filter with the timestamp from
+  `PinStatus.created` found in the oldest item in the current batch of results.
+  Repeat to read all results.
+
+- Returned results can be fine-tuned by applying optional `after`, `cid`,
+  `status` or `meta` filters.
+
```

It might be useful to note the caveat of bulk creation in the spec for the future
```suggestion

> **Note**: pagination by the `created` timestamp assumes no bulk create
> operations. Any future considerations to add support for bulk creation must
> account for this.
```